docs/testing.md

```diff
@@ -0,0 +1,92 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [E2E testing](#e2e-testing)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+### E2E testing
+
```

Comment:
#### Testing with make e2e

I would explain in this section how to use make e2e etc... in it's different flavours, we could move docs from other places to here.. may be the kind scripts directory already has it.

Reply:
There is a very good readme in kind-e2e dir which explains all of this. I have referenced it at many places in these docs.

Reply:
Exactly, I mean we should move those bits *here* because they don't make